benches/Cargo.toml

```diff
@@ -4,6 +4,19 @@ version = "0.1.0"
 edition = "2018"
 workspace = ".."
 
+[features]
+## Enables support for rust-libxml for benchmarks. rust-libxml requires native 
```

Comment:
Single `#` is fine.

Reply:
Fixed with 788bcc8ae92cce1e1c940ab88b7da2966fd38a05